Assets/Scripts/ResourceLogic.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.UIElements;
+using UnityEngine;
+
+public class ResourceLogic : MonoBehaviour
+{
+    //values are out of 100
+    private int paperSupply;
+    private int citizenHappiness;
+    private int money;
+    //value is out of 5 with a HIGHER VALUE = SHORTER GAME!!
+    private int lengthOfGame;
+    private Boolean hasSetLength;
+
+
+    // Start is called before the first frame update
+    void Awake()
+    {
+        paperSupply = 50;
+        citizenHappiness = 50;
+        money = 50;
+        lengthOfGame = 2;
+        hasSetLength = false;
```

Comment:
Is there a reason we are initializing these here instead of where they are declared? (above)

Reply:
I was treating it like a constructor. Not exactly sure if that is what it should be used for though, so this could not be best practice in unity. Would you recommend I change it?